### PR TITLE
Jfratini antioch

### LINKED ISSUE #7: jfratini planning for Antioch file
I want to create a file with:
- [ ] - places to eat
- [ ] - things to do

### PR DESCRIPTION
**Pull request by jfratini**
This pull request has a file with things to do in Antioch.

This PR fixes #7

### CODE REVIEW

**Comment by sajjad2003** [comment]
reviewed

File: jfratini-antioch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Antioch
+
+### Places to eat
```
Comment (sajjad2003) [inline jfratini-antioch.md:3]: i'm your neighbor from gurnee

Comment (sajjad2003) [inline jfratini-antioch.md:3]: reviewed

**Review by sajjad2003 — COMMENTED** [review]
done

File: jfratini-antioch.md (hunk 1 above)
Comment (jfratini) [reply jfratini-antioch.md:3]: :wave:

**Pull request merged**